src/containers/Home/Home.jsx

```diff
@@ -3,33 +3,36 @@ import { Segment, Button, Dropdown, Icon } from 'semantic-ui-react';
 import TabBar from '../../components/TabBar/TabBar';
 
 class Home extends Component {
-    state = {}
+    state = {
+        tabBarName: localStorage.getItem("username"),
+        textBool: true
```

Comment:
Wait if `textBool` is being sent as a prop from the `App` container, why is this inside the state?